bootstrapvz/common/tasks/grub.py

```diff
@@ -61,10 +99,14 @@ def run(cls, info):
 				device_map.write('(hd0) {device_path}\n'.format(device_path=device_path))
 				if not isinstance(p_map, partitionmaps.none.NoPartitions):
 					for idx, partition in enumerate(info.volume.partition_map.partitions):
-						device_map.write('(hd0,{prefix}{idx}) {device_path}\n'
-						                 .format(device_path=partition.device_path,
-						                         prefix=partition_prefix,
-						                         idx=idx + 1))
+						device_map.write(
+							'(hd0,{prefix}{idx}) {device_path}\n'
+							.format(
+								device_path=partition.device_path,
+								prefix=partition_prefix,
+								idx=idx + 1
+							)
+						)
```

Comment:
> Ok, so I noticed the tox.ini file. Is this still being run by the TravisCI build?

Yes. I can [see that the test run finished with][1] `The command "tox" exited with...`.

> Ok, I think I understand... it's a very odd convention that I've never seen.

I had also never seen it before starting to contribute with bootstrap-vz a couple years ago. :-)

Now that you understood, can you please do this on the parts of your PR that are aligned with tabs? If you don't, there's no problem, but I'll have to do this after the merge, and there will be my name on the blame.

> In the future, to make it easier for others to contribute code, you should publish a flake8 config file to detect this.

I don't know if flake8 currently supports this check, so I'll have to look onto it.

[1]: https://travis-ci.org/andsens/bootstrap-vz/builds/106044377